tasks/src/main/scala/dagr/tasks/picard/RevertSam.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package dagr.tasks.picard
+
+import dagr.core.execsystem.{Cores, Memory}
+import dagr.tasks.DagrDef._
+import htsjdk.samtools.SAMFileHeader.SortOrder
+
+import scala.collection.mutable.ListBuffer
+
+class RevertSam(val in: PathToBam,
+                val out: PathToBam,
+                sortOrder: SortOrder = SortOrder.queryname,
+                restoreOriginalQualities: Boolean = true,
+                removeDuplicateInformation: Boolean = true,
+                removeAlignmentInformation: Boolean = true,
+                attributesToClear: Option[Seq[String]] = None,
```

Comment:
I get why you did this, but `Option[Seq[String]]` feels weird.  How would you feel about making it just `Seq[String]` with a default value of `new picard.sam.RevertSam().ATTRIBUTES_TO_CLEAR.toSeq`?